packages/ui/src/components/va-input/components/VaInputWrapper.vue

```diff
@@ -85,6 +71,7 @@
       </div>
     </div>
 
+    <!-- TODO: Can we move it out? -->
```

Comment:
To remove extra div we can do this:
```suggestion
    <template v-if="hasMessages">
      <slot name="messages" v-bind="{ messages: messagesComputed, errorLimit, color: messagesColor }">
        <va-message-list
          class="va-input-wrapper__message-list-wrapper"
          :color="messagesColor"
          :model-value="messagesComputed"
          :limit="errorLimit"
        />
      </slot>
    </template>
```

Reply:
I'm thinking about having 
```vue
<template>
   <flied />
   <messages />
</template>
```

Since we are able to have two or more root elements, we can do this trick.